server/storage/files/S3Storage.ts

```diff
@@ -47,7 +47,7 @@ export default class S3Storage extends BaseStorage {
     );
   }
 
-  public getPublicEndpoint(isServerUpload?: boolean) {
+  public getUploadUrl(isServerUpload?: boolean) {
```

Comment:
Remind me why this change? It seems in the end that it would still better be called `getPublicEndpoint` – you're using it as part of `getUrlForKey` after all?